crates/voicevox_core/src/infer/runtimes/onnxruntime.rs

```diff
@@ -396,6 +401,7 @@ pub(crate) mod blocking {
         /// 一度成功したら以後は同じ参照を返す。
         #[cfg(feature = "link-onnxruntime")]
         #[cfg_attr(docsrs, doc(cfg(feature = "link-onnxruntime")))]
+        #[doc(alias = "voicevox_onnxruntime_init_once")]
```

Comment:
ここだけ上じゃなくて下に書いてそう（別に問題はないけど）
```suggestion
        #[doc(alias = "voicevox_onnxruntime_init_once")]
        #[cfg(feature = "link-onnxruntime")]
        #[cfg_attr(docsrs, doc(cfg(feature = "link-onnxruntime")))]
```